dev/com.ibm.ws.jpa_spec10_query_fat/test-applications/svlquery/ddl/WEB-INF/classes/JPA_SVLQUERY_POPULATE_DB2.ddl

```diff
@@ -45,7 +45,7 @@ INSERT INTO ${schemaname}.JPASUPPLIERPARTTAB_JPAPARTTAB (SUPPLIERS_SID,SUPPLIES_
 INSERT INTO ${schemaname}.JPASUPPLIERPARTTAB_JPAPARTTAB (SUPPLIERS_SID,SUPPLIES_PARTNO) VALUES (1,10);
 INSERT INTO ${schemaname}.JPASUPPLIERPARTTAB_JPAPARTTAB (SUPPLIERS_SID,SUPPLIES_PARTNO) VALUES (1,11);
 INSERT INTO ${schemaname}.JPASUPPLIERPARTTAB_JPAPARTTAB (SUPPLIERS_SID,SUPPLIES_PARTNO) VALUES (1,12);
-INSERT INTO ${schemaname}.JPATASKBEAN (TASKID,COST,DESCRIPTION,NAME,PROJECT_PROJID) VALUES (1010,123.45,'Design ','Design ',1000);
+INSERT INTO ${schemaname}.JPATASKBEAN (TASKID,COST,DESCRIPTION,NAME,PROJECT_PROJID) VALUES (1010,123.45,'Design','Design',1000);
```

Comment:
I think this change should be also made in the HSQL/INFORMIX/MYSQL/SYBASE DDL files for SQLQuery